docs/build/html/_sources/gui.layers.rst.txt

```diff
@@ -0,0 +1,61 @@
+gui.layers package
+==================
+
+Subpackages
+-----------
+
+.. toctree::
+
+    gui.layers.qt
+
+Submodules
+----------
+
+gui.layers.\_base\_layer module
```

Comment:
File names with one or more leading underscores should be ignored.

Reply:
not sure about hiding all filenames with a leading `_` - or maybe some of our files that have a `_` at the beginning really shouldn't - i.e. `_image_layer.py` and `_markers_layer.py` in the main `layers` folder. I think these should be easily accessible to users as they contain methods that users will interact with, right?

It would also be nice if there was a way to see the files with `_` even if just as part of "developer" docs

Reply:
@kne42 @sofroniewn . These are not meant to be docs for users, IMO. Do you guys want to generate a doc for users? I think we should include all for specs-like docs for developers but should exclude if this will be targeting users.

Reply:
I'm not really sure about the division between user docs / dev docs. If I go to something like https://pytorch.org/docs/stable/torch.html do I only see user docs? It looks pretty comprehensive.

Reply:
seems like things are quite complex right now and there's a lot going on - so i'd maybe lean towards more of the "user" - and not include the `_` files directly or the `vispy` stuff, but then maybe we realize there is certain stuff not present that is important and we need to move over

Reply:
It is targeting users. That example docs doesnt give any implication about design choices/implementation. Showing hidden elements/backports is not required for such usage. It is comprehensive but only giving details for usage of software/API.

Reply:
thanks @AhmetCanSolak - i'd then say we focus on users right now, but open to opinions of others.